env.default

```diff
@@ -61,8 +61,9 @@ CSP_FRAME_ANCESTORS=" 'none' "
 CSP_FRAME_SRC=" 'self' https://www.youtube.com https://comments.mozillafoundation.org/ https://airtable.com https://docs.google.com/ https://platform.twitter.com https://public.zenkit.com https://calendar.google.com https://www.youtube-nocookie.com https://form.typeform.com https://js.tito.io https://datawrapper.dwcdn.net"
 CSP_IMG_SRC=" * data: "
 CSP_MEDIA_SRC=" 'self' data: https://s3.amazonaws.com/mofo-assets/foundation/video/ "
-CSP_SCRIPT_SRC=" 'self' 'unsafe-inline' https://www.google-analytics.com/analytics.js http://*.shpg.org/ https://comments.mozillafoundation.org/ https://airtable.com https://platform.twitter.com https://cdn.syndication.twimg.com https://embed.typeform.com https://www.googletagmanager.com https://js.tito.io"
-CSP_STYLE_SRC=" 'self' 'unsafe-inline' https://code.cdn.mozilla.net https://fonts.googleapis.com https://platform.twitter.com https://js.tito.io "
+CSP_SCRIPT_SRC=" 'self' 'unsafe-inline' https://www.google-analytics.com/analytics.js http://*.shpg.org/ https://comments.mozillafoundation.org/ https://airtable.com https://platform.twitter.com https://cdn.syndication.twimg.com https://embed.typeform.com https://js.tito.io https://tagmanager.google.com *.googletagmanager.com"
+CSP_STYLE_SRC=" 'self' 'unsafe-inline' https://code.cdn.mozilla.net https://fonts.googleapis.com https://platform.twitter.com https://js.tito.io https://tagmanager.google.com"
+CSP_INCLUDE_NONCE_IN=script-src
 
 
```

Comment:
connect-src seems to be missing as well.

again, pattern matching:

`CSP-CONNECT-SRC=" *.google-analytics.com *.analytics.google.com *.googletagmanager.com "`

Reply:
Hi Tyler! CSP_CONNECT_SOURCE can be found [here](https://github.com/mozilla/foundation.mozilla.org/blob/6bd998f45484bc21beb7e1b2416e589eb6579b72/env.default#L57), and the default is set to allow everything `" * "`. I double checked the staging and prod instance and found that they were also set to `*` which would allow the 3 URLs mentioned above